CHANGELOG.md

```diff
@@ -56,11 +58,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [0.1.0] - 2019-09-24
 ### Initial release to rubygems.org
 
-[Unreleased]: https://github.com/ManageIQ/manageiq-api-common/compare/v2.0.1...HEAD
-[2.0.1]: https://github.com/ManageIQ/manageiq-api-common/compare/v2.0.0...v2.0.1
-[2.0.0]: https://github.com/ManageIQ/manageiq-api-common/compare/v1.1.0...v2.0.0
-[1.1.0]: https://github.com/ManageIQ/manageiq-api-common/compare/v1.0.2...v1.1.0
-[1.0.2]: https://github.com/ManageIQ/manageiq-api-common/compare/v1.0.1...v1.0.2
-[1.0.1]: https://github.com/ManageIQ/manageiq-api-common/compare/v1.0.0...v1.0.1
-[1.0.0]: https://github.com/ManageIQ/manageiq-api-common/compare/v0.1.0...v1.0.0
-[0.1.0]: https://github.com/ManageIQ/manageiq-api-common/releases/tag/v0.1.0
+[Unreleased]: https://github.com/RedHatInsights/insights-api-common-rails/compare/v2.0.1...HEAD
+[2.0.1]: https://github.com/RedHatInsights/insights-api-common-rails/compare/v2.0.0...v2.0.1
+[2.0.0]: https://github.com/RedHatInsights/insights-api-common-rails/compare/v1.1.0...v2.0.0
+[1.1.0]: https://github.com/RedHatInsights/insights-api-common-rails/compare/v1.0.2...v1.1.0
+[1.0.2]: https://github.com/RedHatInsights/insights-api-common-rails/compare/v1.0.1...v1.0.2
+[1.0.1]: https://github.com/RedHatInsights/insights-api-common-rails/compare/v1.0.0...v1.0.1
+[1.0.0]: https://github.com/RedHatInsights/insights-api-common-rails/compare/v0.1.0...v1.0.0
```

Comment:
In regards to the CHANGELOG comments earlier, possibly I'm misunderstanding the `[Unreleased]` heading.  I would have expected line 67 to still say `ManageIQ` since on line 39 it was released before this name change PR. However if the `[Unreleased]` heading needs the current 'real' name/location of the repo then these changes make sense.

Reply:
These lines at the bottom are links and when we move the repo they will no longer be valid.  I have to change these.  What I was referring to were references to ManageIQ::Api constants elsewhere in the content, and those I left alone.

Reply:
Should the change to `RedHatInsights` happen after the transfer?

Reply:
I think it can go ahead of time...it's just docs and it's ok if they temporarily don't resolve correctly.